webpack.config.js

```diff
@@ -247,6 +247,7 @@ var config = {
             'mattermost-redux/test': 'packages/mattermost-redux/test',
             'mattermost-redux': 'packages/mattermost-redux/src',
             reselect: 'packages/reselect/src',
+            '@mui/styled-engine': '@mui/styled-engine-sc',
```

Comment:
Why do we need an alias for this? I would've expected this to just work out of the box without any sort of import aliasing

Reply:
Yeah material ui works out of box for emotion-css. To make it work with styled components we need this alias. https://mui.com/material-ui/guides/styled-engine/

Reply:
Hopefully this won't break when we start to using the components package elsewhere. We'll probably need to look at how Babel handles this, but that can be done later when we move this into the components package.

CC @michelengelen